docker-compose.yml

```diff
@@ -14,7 +14,21 @@ services:
     volumes:
       - "/srv/hommod:/data"
 
-  celery:
+  celery-beat:
```

Comment:
Why do you have a separate container for celery beat instead of using the existing worker container and passing `-B`?